src/GraphQL.Primitives/GraphQLRequest.cs

```diff
@@ -49,6 +49,14 @@ public GraphQLRequest(string query, object? variables = null, string? operationN
             OperationName = operationName;
         }
 
+        public GraphQLRequest(IEnumerable<KeyValuePair<string, object>> values)
```

Comment:
This constructor seems a bit odd, but so does the fact that `GraphQLRequest` inherits from a `Dictionary<>`...

I would suggest this change:
```suggestion
        public GraphQLRequest(GraphQLRequest other)
```
The `foreach` code below should still be valid.

Reply:
If someone actually needed to clone a generic dictionary into a `GraphQLRequest` class, I'd say they write a `foreach` snippet themselves, but this would not be typical.  Correct???  I've only used this library a few times, so lmk if not.

Reply:
Sounds reasonable. You're right that `IEnumerable<KeyValuePair<string, object>>` is more generic than it needs to be.

Reply:
Fixed copy constructor.